### PR TITLE
Cleaning up old cruft

### PR DESCRIPTION
**Pull request by coolaj86**
Re: https://github.com/go-gitea/gitea/issues/5066

* [x] Remove dangling reference to `gorilla/context`
* [x] Remove gorilla from vendor
* [x] Remove gorilla from Gopkg.*
* [x] Remove old gogs script
* [x] Remove old mysql script

### CODE REVIEW

**Comment by coolaj86** [comment]
@lafriks Was the `gorilla/context` added for `xormstore` and/or `goth`?

**Comment by coolaj86** [comment]
@bkcsoft this isn't ready for lgtm - it's passing test due to gorilla being used by xormstore (and goth?). The thing I removed may actually be a necessary hack to prevent memory leaks in the oauth2 flow. Awaiting feedback.

**Comment by adelowo** [comment]
> @bkcsoft this isn't ready for lgtm 

I think the lgtm label thing is automated

File: cmd/web.go [hunk 1]
```diff
@@ -170,23 +169,23 @@ func runWeb(ctx *cli.Context) error {
...
 		}
-		err = runHTTPS(listenAddr, setting.CertFile, setting.KeyFile, context2.ClearHandler(m))
+		err = runHTTPS(listenAddr, setting.CertFile, setting.KeyFile, m)
 	case setting.FCGI:
 		listener, err := net.Listen("tcp", listenAddr)
 		if err != nil {
 			log.Fatal(4, "Failed to bind %s", listenAddr, err)
 		}
 		defer listener.Close()
-		err = fcgi.Serve(listener, context2.ClearHandler(m))
```
Comment (techknowlogick) [inline cmd/web.go:189]: `context2.ClearHandler()`  is used for to prevent memory leaks. I think it may still be needed.

**Pull request closed without merging**